Misc tutorial improvements (#2006)
kframework/k: k-distribution/k-tutorial/1_basic/07_side_conditions/README.md

rules, and will only be tried to be applied after all the other,
higher-priority rules have been tried and they have failed.

For example, in the above exercise, we had to add a side condition containing
two boolean comparisons to the rule we wrote to handle `B()` grades. However,
in practice this meant that we compare the percentile to 90 twice. We can more
efficiently and more idiomatically write the `B()` case for the
`gradeFromPercentile` rule using the `owise` attribute (`lesson-07-b.k`):

```k
module LESSON-07-B
  imports BOOL
  imports INT

  syntax Grade ::= A() | B() | C() | D() | F()
                 | gradeFromPercentile(Int) [function]

  rule gradeFromPercentile(I) => A() requires I >=Int 90
  rule gradeFromPercentile(I) => B() requires I >=Int 80 [owise]
endmodule
```

> This rule is saying, "if the first rule does not apply, then the grade is a B
> if the percentile is greater than or equal to 80." Note here that we use both

Reviewer: ```suggestion
This rule is saying, "if the all other rules which match do not apply, then the grade is a B
if the percentile is greater than or equal to 80." Note here that we use both
```

a side condition and an `owise` attribute on the same rule. This is not
required (as we will see later), but it is allowed. What this means is that the

> side condition is only tried if the previous rules did not apply **and** the 

Reviewer: ```suggestion
side condition is only tried if the other rules did not apply **and** the 
```

I would prefer to say "other" instead of "previous" in these cases, because "previous" might imply that it means "earlier in the file", which we do not mean to imply.
